app/templates/core/subscribe-modal.jade

```diff
@@ -14,101 +14,73 @@
-            if !me.isOnPremiumServer()
-              td.center-ok.free-cell
-                span.glyphicon.glyphicon-ok
-            td.center-ok
-              span.glyphicon.glyphicon-ok
-          tr
-            td.feature-description
-              span(data-i18n="[html]subscribe.feature3", data-i18n-options='{"bonusLevelsCount": ' + view.i18nData.bonusLevelsCount + '}'))
-            if !me.isOnPremiumServer()
-              td.free-cell
-            td.center-ok
-              span.glyphicon.glyphicon-ok
-          // Could try testing out things like features 6-8 in a new design with more space
-          //tr
-          //  td.feature-description
-          //    span(data-i18n="[html]subscribe.feature7")
-          //  if !me.isOnPremiumServer()
-          //    td.free-cell
-          //  td.center-ok
-          //    span.glyphicon.glyphicon-ok
-          //tr
-          //  td.feature-description
-          //    span(data-i18n="subscribe.feature6")
-          //  if !me.isOnPremiumServer()
-          //    td.free-cell
-          //  td.center-ok
-          //    span.glyphicon.glyphicon-ok
-          //if me.getCampaignAdsGroup() === 'leaderboard-ads'
-          //  tr
-          //    td.feature-description
-          //      span(data-i18n="[html]subscribe.feature8")
-          //    if !me.isOnPremiumServer()
-          //      td.free-cell
-          //    td.center-ok
-          //      span.glyphicon.glyphicon-ok
-          tr
-            td.feature-description
-              span(data-i18n="[html]subscribe.feature_game_dev")
-            if !me.isOnPremiumServer()
-              td.free-cell
-            td.center-ok
-              span.glyphicon.glyphicon-ok
-          tr
-            td.feature-description
-              span(data-i18n="[html]subscribe.feature_web_dev")
-            if !me.isOnPremiumServer()
-              td.free-cell
-            td.center-ok
-              span.glyphicon.glyphicon-ok
-          tr
-            td.feature-description
-              span(data-i18n="[html]subscribe.feature2", data-i18n-options='{"heroesCount": ' + view.i18nData.heroesCount + '}')
-            if !me.isOnPremiumServer()
-              td.free-cell
-            td.center-ok
-              span.glyphicon.glyphicon-ok
-          tr
-            td.feature-description.gem-amount
-              span(data-i18n="[html]subscribe.feature4")
-            if !me.isOnPremiumServer()
-              td.free-cell
-            td.center-ok
-              span.glyphicon.glyphicon-ok
+      div.paper-area
+        div.benefits-header.text-center(data-i18n="[html]subscribe.comparison_blurb")
+          
+            
+        .container-fluid
+          .row
+            .col-xs-5.feature-col.col-xs-offset-1
+              ul
+                li 
+                  ="• "
```

Comment:
Random unicode characters here seems dangerous.  It also mixes style with our presentation.  Could we move the bullets to the css with ::before ?